workerapi/server/setup.go

```diff
@@ -43,19 +47,27 @@ func Defaults() (*ice.MagicBag, jsonconfig.Schema) {
 		endpoints.MakeStatsReceiver,
```

Comment:
Actually while we're here I'm realizing that I greatly prefer when we explicitly state the inputs/outputs for each func. Do you think it's too noisy to write functions that delegate to something with the same signature?